setup.cfg

```diff
@@ -28,11 +27,7 @@ install_requires =
 [options.extras_require]
 test =
     pytest-astropy
-    pytest-cov
-    tox
```

Comment:
I just added `tox` here so that people can run `pip install -e .[test]` locally and get everything they need to check their code locally before opening a PR, e.g. running `tox -e codestyle` (or any of the other tests via `tox`). I would prefer to keep this in.

Reply:
Okay but you don't really need tox to do that. You can run `flake8 specutils` locally and it is the same check.